internal/resource/statefulset.go

```diff
@@ -614,7 +614,7 @@ func (builder *StatefulSetBuilder) podTemplateSpec(previousPodAnnotations map[st
 								Command: []string{"/bin/bash", "-c",
 									fmt.Sprintf("if [ ! -z \"$(cat /etc/pod-info/%s)\" ]; then exit 0; fi;", DeletionMarker) +
 										fmt.Sprintf(" rabbitmq-upgrade await_online_quorum_plus_one -t %d &&"+
-											" rabbitmq-upgrade await_online_synchronized_mirror -t %d &&"+
+											" rabbitmq-upgrade await_online_synchronized_mirror -t %d || true &&"+
```

Comment:
Does this actually do what we expect? There's an `&&` following the `|| true`. I think we need to somehow group `rabbitmq-upgrade await_online_synchronized_mirror || true` into one command, so that the logic works out right.

Reply:
Apparently, not in Bash or other shells. TIL
https://unix.stackexchange.com/questions/88850/precedence-of-the-shell-logical-operators#:~:text=In%20many%20computer%20languages%2C%20operators,%7C%7C%20have%20the%20same%20precedence.

Reply:
Yeah, I kind of expected this not to work but I did try. Information about the node going into maintenance mode is back. Also, a manual tests like this:
```
bash -c 'echo foo && ehco bar || echo meh && echo baz'
foo
bash: line 1: ehco: command not found
meh
baz
```